src/pykeen/utils.py

```diff
@@ -203,7 +206,7 @@ def _flatten_dictionary(
 
 
 def get_embedding_in_canonical_shape(
```

Comment:
This function is never called in PyKEEN with `ind=None`. Is there a reason it has this functionality implemented?